package.json

```diff
@@ -27,8 +27,8 @@
   },
   "homepage": "https://github.com/ng-bootstrap/ng-bootstrap#readme",
   "dependencies": {
-    "@angular/core": "2.3.1",
     "@angular/common": "2.3.1",
+    "@angular/core": "2.3.1",
```

Comment:
Why this change? While it doesn't hurt it always best idea to avoid unrelated changes in a single PR.